data/kubemacpool/002-configmap.yaml

```diff
@@ -1,8 +1,8 @@
 ---
 apiVersion: v1
 data:
-  START_POOL_RANGE: {{ .StartPoolRange }}
-  END_POOL_RANGE: {{ .EndPoolRange }}
+  START_POOL_RANGE: {{ .StartRange }}
```

Comment:
I think we should also update the https://github.com/K8sNetworkPlumbingWG/kubemacpool and change it to `RANGE_START` 

@phoracek @dankenigsberg  what you think?

Reply:
I woudn't mind. There won't be a better time to do that than now.

Reply:
ack for `RANGE_START`. We should try to use proper English, and should avoid the object name if it is obvious from context.